### ISSUE TITLE
New configuration options: enabled, crossorigin

### ISSUE THREAD

**Issue by jscheid** [waysact/webpack-subresource-integrity#18]
It should be possible to disable the plugin (e.g. in development mode) and configure the `crossorigin` attribute for html-webpack-plugin injection. To this end, the current configuration syntax is now deprecated and will be replaced by a standard key/value based option argument.

The old plugin initialization looks like this:

```javascript
new SriPlugin(['sha256', 'sha384']) // multiple hash function names, or
new SriPlugin('sha512') // single hash function name
```

This is replaced by:

```javascript
new SriPlugin({
    hashFuncNames: ['sha512'],
})
```

Note that a single hash function name also needs to be wrapped in an array.

The new syntax is supported from 0.7.0 onwards. The old configuration syntax is still supported but will cause a deprecation warning to be output via Webpack facilities, and will be removed in a future 1.0.0 release.

This change introduces the `enabled` option, which users should consider setting to a falsy value outside of production. It also introduces the `crossorigin` option, which users will want to leave at its default value (`anonymous`) in most cases. See the new README for details.

**Comment by jscheid** [comment]
This change was released in v0.7.0.

**Comment by pwnsdx** [comment]
Good move! :+1:
